### PR TITLE
correct occ command for mimetypealiases

### PR DESCRIPTION
**Pull request by carlaschroder**
Applies to 8.2+

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by rullzer** [comment]
:+1:

File: resources/config/mimetypealiases.dist.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	"_comment3": "Put any custom mappings in a new file mimetypealiases.json in the config/ folder of ownCloud",
 
 	"_comment4": "After any change to mimetypealiases.json run:",
-	"_comment5": "./occ maintenance:mimetypesjs",
+	"_comment5": "./occ occ maintenance:mimetype:update-js",
```
Comment (Xenopathic) [inline resources/config/mimetypealiases.dist.json]: `./occ occ`? Typo there?

Comment (rullzer) [inline resources/config/mimetypealiases.dist.json]: yes... damn my eyes skipped right over that the first time...

Comment (DeepDiver1975) [inline resources/config/mimetypealiases.dist.json]: yes :see_no_evil:

**Comment by carlaschroder** [comment]
augh. Stupid keyboard. It's always the keyboard's fault. Fixed.

**Comment by carlaschroder** [comment]
Oh phooey, it's already merged.

**Comment by carlaschroder** [comment]
new pr to fix typo https://github.com/owncloud/core/pull/24290

**Comment by MorrisJobke** [comment]
@rullzer Could I ask you to backport this to stable8.2?

@karlitschek Is the backport of this comment okay?

**Comment by karlitschek** [comment]
nice. backport is a nobrainer 👍

**Pull request merged**